third_party/purge/acc.purge.php

```diff
@@ -63,7 +66,17 @@ public function process_purge_request()
 			
 			foreach ($urls as $url)
 			{
-				send_purge_request($url, $port);
+				if($_POST['purge_url'] != '')
+					$_url = preg_replace('/\/$/','',$url).'/'.preg_replace('/^\//','',$_POST['purge_url']); //handle trailing and beginning slashes
+				else 
+					$_url = $url;
+				$resp = send_purge_request($_url, $port);
+				
+				echo $resp; //depending on curl to send varnish's response or an error
+				
+				die; //we die here instead of return to cut of template debugging 
```

Comment:
Hmm hang on a second. Since you die inside the loop, this no longer supports purging multiple servers specified via the `varnish_site_url` config. It'll only do one. Is there another way to handle this?